custom_dc/quickstart.md

```diff
@@ -33,7 +33,7 @@ The "services" Docker container consists of the following Data Commons component
 
 ## Prerequisites
 
-- Obtain a [GCP](https://console.cloud.google.com/welcome) billing account and project.
+- Obtain a [GCP](https://cloud.google.com/docs/get-started) billing account and project.
```

Comment:
Can we ensure this link and all external links open in a new browser?

Reply:
So Prem only mentioned Console links. Do you think we should make all external doc links open in a new window?

Reply:
Yes.

Reply:
OK, let me do that in another PR, because it will affect the entire doc set.